azkaban-db/src/main/sql/upgrade.3.86.0.to.3.87.0.sql

```diff
@@ -0,0 +1,6 @@
+-- DB Migration from release 3.86.0 to 3.87.0
+-- Expand the length of fields related to ramp feature
+ALTER TABLE ramp_dependency CHANGE COLUMN dependency dependency VARCHAR(200) NOT NULL , CHANGE COLUMN defaultValue defaultValue VARCHAR(1000) NULL DEFAULT NULL;
+ALTER TABLE ramp_items CHANGE COLUMN dependency dependency VARCHAR(200) NOT NULL , CHANGE COLUMN rampValue rampValue VARCHAR(1000) NULL DEFAULT NULL;
```

Comment:
Here `rampValue` column is set to NULL, however in the create sql file is NOT NULL. Can we make constraints the same in both places?

Reply:
thanks. good catch.